### DISCUSSION TITLE
Adding Tunables to Debug > Saved Tunables List

### DISCUSSION THREAD

**Discussion by OXP3573** [YimMenu/YimMenu#3537] [General]
With the new tunables debug option, changing values mid-session works fine. Once you change session, tuneables_processing script restores the network values which is annoying. 

I assume the saved list, once populated pushes the chosen tunable values to memory repeatedly to retain it... what should we do to create a list of tunables and the respective value to add to the saved list? I cannot see any files in the Yim folder to populate the list.

**Comment by ShinyWasabi** [comment]
When you load a saved tunable it will show you its current value, not the value at the time you saved it.

You can set `iLocal_65` in social_controller to 0 in a loop to prevent tuneables_processing from being started.

**Reply by OXP3573** [reply]
Would you mind listing some steps to loop this please? I also cannot see an option to save a tunable in the menu or a file in the config folder to add them.

I removed tuneables_processing from the script_rel.rpf and it seems to allow me to change variables that stick with a session change. It's only a plus, network tunables contain suspicious sigs at the bottom.

**Reply by ShinyWasabi** [reply]
```lua
script.register_looped("Block tuneables_processing", function()
    locals.set_int("social_controller", 65, 0)
end)
```
Save this as a lua file and put it in the scripts folder.

**Reply by OXP3573** [reply]
> ```lua
> script.register_looped("Block tuneables_processing", function()
>     locals.set_int("social_controller", 65, 0)
> end)
> ```
> 
> Save this as a lua file and put it in the scripts folder.

Great thanks.

I was thinking the closest we could get to serving a custom tunables to the game without needing to manually change variables each game load is to potentially hook the debug function with an LUA containing a preset list of tunables and the respective custom values. How would one go about doing that, would you be able to create a template with one or two to expand on? Thanks.
